core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -510,6 +510,7 @@ export namespace RealityModelTileTree {
     name?: string;
     classifiers?: SpatialClassifiers;
     planarClipMask?: PlanarClipMaskSettings;
+    scalablemeshProps?: any;
```

Comment:
I think we should enforce the type and not use "any" here if possible.

Reply:
Done! See 3f8952f